src/components/Header/NavBar.tsx

```diff
@@ -5,6 +5,8 @@ import { getNavigationProps } from "./getNavigationProps"
 import { useNavBar } from "./useNavBar/useNavBar"
 import styles from "./scroll.module.css"
 
+declare const Weglot: any
```

Comment:
Without this TS check fails because doesn't know of the global Weblog variable.

Reply:
LGTM